flojoy/plotly_utils.py

```diff
@@ -31,6 +31,8 @@ def data_container_to_plotly(data: DataContainer) -> dict[str, Any]:
             fig = px.scatter_3d(x=data_copy.x, y=data_copy.y, z=data_copy.z)
         case "scalar":
             fig = px.histogram(x=data_copy.c)
+        case "vector":
```

Comment:
I don't think a histogram is a great way to represent a vector, maybe you could make it a table with 1 column like with dataframe? I would recommend cutting it off after a certain amount of elements though, like if you had a vector with 1000 elements then just display something like `[1,2,3,4,5,6,7,8,...,1000]`

Reply:
Why should we skip them instead of just displaying them all? The dataFrame type is displaying all the values despite its length.